launch.json

```diff
@@ -10,7 +10,7 @@
       "request": "launch",
       "cwd": "${workspaceRoot}/loveworks/src/api",
       "runtimeExecutable": "npm",
-      "runtimeArgs": ["run-script", "offline"],
+      "runtimeArgs": ["run-script", "TODO"],
```

Comment:
my current install has the left one as Debug API (old) and my current version has       "runtimeArgs": ["index.ts"],
I confess I don't quite see why you're passing "TODO" as a runtimeArg here, I assume it does nothing but it wasn't obvious to me that was so.

Reply:
The `offline` script was a reference to the command called through in to the serverless framework to run the API locally.  That was invalidated when we switched to CDK due to the now resolved SSO integration.  However, It hasn't been a priority to replace it since we have the type-safe-plus-schema based extraction of events and separation of the logic for handling requests.  Those practices leave us in a great position to run unit tests for meaningful validation of most of that code.

That's the history and reason we have yet to prioritize replacement.  The TODO is to avoid false confidence in the meaningfulness of the launch profile while leaving most of the stub for any future desired use.